lib/Listerine/channels.ex

```diff
@@ -165,4 +165,25 @@ defmodule Listerine.Channels do
         end).()
     |> prepend.(remove_course_channels(others))
   end
+  
+  def role_list(_message, [], _mode) do
+  end 
+
+  def role_list(message, [name | tail], mode) do
```

Comment:
Function names should start with a verb. I have no idea what this might do based on it's name.